static/js/stream_ui_updates.js

```diff
@@ -70,8 +70,22 @@ export function update_toggler_for_sub(sub) {
         }
         stream_edit.toggler.disable_tab("personal_settings");
     }
+    update_toggler_for_sub_private_settings(sub);
```

Comment:
We can call this only if stream is private and remove the check below in the function.

Reply:
Also, we can rename the function to something else to be more clear like `enable_or_disable_subscribers_tab` or something better.